test/tools/Modules/WebListener/WebListener.psm1

```diff
@@ -140,9 +154,15 @@ function Get-WebListenerUrl {
         $Uri.Host = 'localhost'
         $Uri.Port = $runningListener.HttpPort
         $Uri.Scheme = 'Http'
+
         if ($Https.IsPresent)
         {
-            $Uri.Port = $runningListener.HttpsPort
+            switch ($SslProtocol)
+            {
+                'Tls11' { $Uri.Port = $runningListener.Tls11Port }
+                'Tls'   { $Uri.Port = $runningListener.TlsPort }
+                default { $Uri.Port = $runningListener.HttpsPort }
```

Comment:
Kestrel default protocol is really Tls12? If so we need comments here and in docs.

Reply:
I see. Please add comments here and in README.md

Reply:
ah.. I did have it in the readme.. it missed the commit. I'll add it

Reply:
fixed.